bundles/org.openhab.binding.sunsynk/src/main/java/org/openhab/binding/sunsynk/internal/api/dto/Client.java

```diff
@@ -39,7 +39,7 @@ public class Client {
     public Client() {
     }
 
-    public String getAccessTokenString() {
+    public static String getAccessTokenString() {
```

Comment:
Need to double check this, but ask myself why is this static? The accesstoken is dynamic by nature as it is valid for a certain time and expires.

Reply:
Removed static